bigbluebutton-html5/imports/api/users/server/eventHandlers.js

```diff
@@ -8,15 +8,13 @@ import handleEmojiStatus from './handlers/emojiStatus';
 import handleChangeRole from './handlers/changeRole';
 import handleUserPinChanged from './handlers/userPinChanged';
 import handleUserInactivityInspect from './handlers/userInactivityInspect';
-import handleChangeMobileFlag from '/imports/api/users/server/handlers/changeMobileFlag';
```

Comment:
lines 11 and 19 are part of https://github.com/bigbluebutton/bigbluebutton/pull/17496